modules/ROOT/pages/getting-started.adoc

```diff
@@ -2,16 +2,17 @@
 
 == Introduction
 
-=== Update Streams
+=== Streams
 
-There are three Fedora CoreOS (FCOS) update streams available: `stable`, `testing`, and `next`. In general, you will want to use `stable`, but it is recommended to run some machines on `testing` and `next` as well.
+Metadata about Fedora CoreOS is available in a custom JSON format, called "stream metadata".  For maintaining automation, it is expected that you will interact with this stream metadata.  There are three Fedora CoreOS (FCOS) update streams available: `stable`, `testing`, and `next`. In general, you will want to use `stable`, but it is recommended to run some machines on `testing` and `next` as well and provide feedback.
```

Comment:
Feels funny to talk about JSON metadata right away on the Getting Started page before talking about the streams more generally.

Maybe let's invert the order of these sentences? And then at that point, we might as well make it work with the following sentence about the canonical URL. So e.g. something like

```suggestion
There are three Fedora CoreOS (FCOS) update streams available: `stable`, `testing`, and `next`. In general, you will want to use `stable`, but it is recommended to run some machines on `testing` and `next` as well and provide feedback.

Each stream has a canonical URL representing its current state in JSON format. This is called the "stream metadata".  For example, the stream metadata URL for `stable` is: https://builds.coreos.fedoraproject.org/streams/stable.json. For maintaining automation, it is expected that you will interact with stream metadata.
```

?